pyrightconfig.json

```
@@ -6,7 +6,8 @@
     "dist",
     "typings",
     "sandbox",
-    "docs"
+    "docs",
+    "tests/deploys"
```

We should try to just ignore the apps. But our test files should be type compliant

```suggestion
    "tests/deploys/apps"
```